src/components/SelectedAnnotation.jsx

```diff
@@ -73,17 +63,22 @@ class SelectedAnnotation extends React.Component {
   }
 
   onKeyDown(e) {
+    if (e.ctrlKey || e.metaKey || e.altKey || e.shiftKey) {
```

Comment:
Not sure what sort of solution we arrived on here, but this works with keeping in copy/paste functionality, etc. It will also allow vowels to be added for users with Hebrew keyboards (although not sure we will necessarily need this).